### PR TITLE
fixed commands list

### PR DESCRIPTION
**Pull request by StimkaT**

### CODE REVIEW

File: projects/app/src/lib/containers/commands-list-container/commands-list-container.component.ts [hunk 1]
```diff
@@ -26,10 +26,10 @@ export class CommandsListContainerComponent {
     private readonly store: Store,
     ) {}
 
-  buttonClick($event: any) {
-    // console.log($event);
-    this.store.dispatch(sendMessage({
-      message: $event
-    }));
+    events($event: any) {
+      console.log($event);
```
Comment (StimkaT) [inline projects/app/src/lib/containers/commands-list-container/commands-list-container.component.ts:30]: comment

File: projects/app/src/lib/containers/commands-list-container/commands-list-container.component.ts [hunk 2]
```diff
@@ -26,10 +26,10 @@ export class CommandsListContainerComponent {
...
+      if ($event.event === 'CommandsListComponent:buttonClick') {
+      this.store.dispatch(sendMessage({ message: $event.data }));
```
Comment (sislex) [inline projects/app/src/lib/containers/commands-list-container/commands-list-container.component.ts:32]: add indent (tab) in line 32

File: projects/blink/src/lib/containers/blink-commands-list-container/blink-commands-list-container.component.ts [hunk 3]
```diff
@@ -24,11 +24,10 @@ export class BlinkCommandsListContainer {
...
-  buttonClick($event: any) {
-    // console.log($event);
-    this.store.dispatch(sendMessageToDevice({message: {
-      event: "LED",
-      data: $event
-    }}));
+    events($event: any) {
+      console.log($event);
+      if ($event.event === 'CommandsListComponent:buttonClick') {
+      this.store.dispatch(sendMessageToDevice({ message: $event.data }));
```
Comment (sislex) [inline projects/blink/src/lib/containers/blink-commands-list-container/blink-commands-list-container.component.ts:30]: add tab

File: projects/blink/src/lib/containers/blink-commands-list-container/blink-commands-list-container.component.ts [hunk 4]
```diff
@@ -24,11 +24,10 @@ export class BlinkCommandsListContainer {
...
     ) {}
 
-  buttonClick($event: any) {
-    // console.log($event);
-    this.store.dispatch(sendMessageToDevice({message: {
-      event: "LED",
-      data: $event
-    }}));
+    events($event: any) {
+      console.log($event);
```
Comment (sislex) [inline projects/blink/src/lib/containers/blink-commands-list-container/blink-commands-list-container.component.ts:28]: comment console

**Pull request merged**